assets/js/payment-method-extensions/payment-methods/stripe/credit-card/payment-method.js

```diff
@@ -13,6 +13,7 @@ import { InlineCard, CardElements } from './elements';
 import { Elements, useElements, useStripe } from '@stripe/react-stripe-js';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { useCheckoutContext } from '@woocommerce/base-context';
```

Comment:
Don't use hooks directly in any of the payment method integrations.  All data is exposed via `use-payment-method-interface`. So the `customerId` will need to be passed through on `usePaymentMethodInterface` (and related type-defs etc will need updated).

Reply:
Can you check again @nerrad?

We should review all data on this interface before making public just to ensure keys etc make sense.